aws/resource_aws_iot_topic_rule_test.go

```diff
@@ -861,6 +886,31 @@ resource "aws_iot_topic_rule" "rule" {
 `, rName, separator)
 }
 
+func testAccAWSIoTTopicRule_kafka(rName string) string {
+	return fmt.Sprintf(testAccAWSIoTTopicRuleRole+`
+resource "aws_vpc" "iot_vpc" {
+	cidr_block = "10.0.0.0/16"
+}
+resource "aws_secretsmanager_secret" "keystore" {
+	name = "keystore"
+}
+resource "aws_iot_topic_rule" "rule" {
+  name        = "test_rule_%[1]s"
+  description = "Example rule"
+  enabled     = true
+  sql         = "SELECT * FROM 'topic/test'"
+  sql_version = "2015-10-08"
+  kafka {
+    destination_arn    = aws_vpc.iot_vpc.arn
+		topic							 = "fake_topic"
+		bootstrap_servers = "b-1.localhost:9094"
+		ssl_keystore 					= "${get_secret("aws_secretsmanager_secret.keystore.arn", "SecretBinary", "", "aws_iam_role.iot_role.arn")}"
```

Comment:
@toddradigan Do you just need to escape this? Is that your problem? Just ping me later if you want to chat more.
```suggestion
		ssl_keystore 					= "$${get_secret("${aws_secretsmanager_secret.keystore.arn}", "SecretBinary", "", "${aws_iam_role.iot_role.arn}")}"
```